include/zeek-compat.h

```diff
@@ -55,24 +41,21 @@
 #include <zeek/Expr.h>
 #include <zeek/IPAddr.h>
 #include <zeek/Reporter.h>
+#include <zeek/Tag.h>
 #include <zeek/Type.h>
 #include <zeek/Val.h>
 #include <zeek/Var.h>
 #include <zeek/analyzer/Analyzer.h>
 #include <zeek/analyzer/Manager.h>
 #include <zeek/analyzer/protocol/pia/PIA.h>
 #include <zeek/analyzer/protocol/tcp/TCP.h>
-#if ZEEK_VERSION_NUMBER < 40100 // Zeek < 4.1
-#include <zeek/Sessions.h>
-#include <zeek/analyzer/protocol/udp/UDP.h>
-#else // Zeek >= 4.1
-#include <zeek/session/Manager.h>
-#endif
```

Comment:
From the diff it's hard to say if some of the `#else` blocks being removed were triggering for Zeek >= 5.0. Please just double-check that nothing goes out accidentally (although then things would presumably break, so probably fine as it is).

Reply:
This should be fine, the include for `zeek/session/Manager.h` stays, the one for `zeek/Sessions.h` is removed. What makes these include changes harder to review is that they also end up resorted (previously the preprocessor calls acted as barriers).